requirements-base.txt

```diff
@@ -56,7 +56,7 @@ six>=1.11.0,<1.12.0
 sqlparse>=0.2.0,<0.3.0
 statsd>=3.1.0,<3.2.0
 structlog==16.1.0
-symbolic>=7.3.1,<8.0.0
+symbolic>=7.3.5,<8.0.0
```

Comment:
Note that after merging we'll have to bump this in `getsentry`.

Reply:
👍 also, it seems you haven’t published this yet :-D

Reply:
It is now.